app/controllers/movies_controller.rb

```diff
@@ -1,4 +1,17 @@
 class MoviesController < ApplicationController
+  def show
+    conn = Faraday.new(url: 'https://api.themoviedb.org') do |f|
+      f.params['api_key'] = ENV['MOVIE_API_KEY']
+    end
+    movie_id = params[:id]
+    movie_stats = conn.get("/3/movie/#{movie_id}")
+    @movie_stats = JSON.parse(movie_stats.body, symbolize_names: true)
+    movie_reviews = conn.get("/3/movie/#{movie_id}/reviews")
+    @movie_reviews = JSON.parse(movie_reviews.body, symbolize_names: true)
+    movie_cast = conn.get("/3/movie/#{movie_id}/credits")
+    @movie_cast = JSON.parse(movie_cast.body, symbolize_names: true)
+  end
+
```

Comment:
refactor this method to have less instance methods and maybe not so many api calls